blade-egui/src/lib.rs

```diff
@@ -144,7 +204,11 @@ impl GuiPainter {
                 ..Default::default()
             },
             depth_stencil: None, //TODO?
-            fragment: shader.at("fs_main"),
+            fragment: if info.format.is_srgb() {
```

Comment:
This looks highly suspicious to me.
Generally speaking, I expect the rendering pipeline to assume linear space render targets all the way. I don't think there is value in expanding this assumption any further.

Reply:
This is what egui-wgpu did so I just replicated all of it to see if it would help.

Egui examples use a non-srg render target whereas blade always uses one

Reply:
Minimizing changes is a good experiment for sure. Shipping it the way egui_wgpu did seems very concerning to me. I'll try to find some associated discussion there.